### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by StarfallProjects**
Need the following:

- [x] Minimum Python version
- [x] Are there a minimum supported Pandas and Spark versions
- [x] Architecture diagram (@keelychamb is providing this)
- [x] Security issue email address
- [x] Newsletter signup (or remove the "stay tuned" bit for now?)
- [x] Roadmap (@kjam )
- [x] Contributors widget (think we need someone with control of the company GitHub account to sign up for it: https://github.com/sourcerer-io/hall-of-fame)

### CODE REVIEW

**Comment by justin1121** [comment]
Can user ```[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/0)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/1)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/2)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/3)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/4)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/5)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/6)[](https://sourcerer.io/fame/justin1121/capeprivacy/cape-python/links/7)```
For the sourcerer hall of fame.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-[<img src="https://raw.githubusercontent.com/dropoutlabs/files/master/cape-logo.png" alt="Cape Privacy" width="500"/>](https://capeprivacy.com/)
+# Cape Python
 
 ![](https://github.com/capeprivacy/cape-python/workflows/Main/badge.svg) 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) 
 [![codecov](https://codecov.io/gh/capeprivacy/cape-python/branch/master/graph/badge.svg?token=L9A8HFAJK5)](https://codecov.io/gh/capeprivacy/cape-python)
 [![PyPI version](https://badge.fury.io/py/cape-privacy.svg)](https://badge.fury.io/py/cape-privacy)
 [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-7A5979.svg)](https://join.slack.com/t/capecommunity/shared_invite/zt-f8jeskkm-r9_FD0o4LkuQqhJSa~~IQA)
 
+A Python library supporting data transformations and collaborative privacy policies, for data science projects in Pandas and Apache Spark
```
Comment (gavinuhma) [inline README.md:9]: I think we should elaborate a bit more in this initial paragraph.

File: README.md [hunk 2]
```diff
@@ -111,17 +110,56 @@ print(df.head())
...
-## Contributing and Bug Reports
+**Upcoming features:**
 
-Please file any [feature request](https://github.com/capeprivacy/cape-python/issues/new?template=feature_request.md) or 
-[bug report](https://github.com/capeprivacy/cape-python/issues/new?template=bug_report.md) as GitHub issues.
+* Reversible tokenisation: allow reversing of tokenization to reveal the raw value.
+* Policy audit logging: create logging hooks to allow audit logs for policy downloads and usage in Cape Python.
+* Expand pipeline integrations: add Apache Beam, Apache Flink, Apache Arrow Flight or Dask integration as another pipeline we can support, either as part of Cape Python or in its own separate project.
+
+The goal is a complete data management ecosystem. Cape Privacy provides [Cape Coordinator](/cape-core/coordinator/), to manage policy and users. This will interact with the Cape Privacy libraries (such as [Cape Python](/libraries/cape-python/)) through a workers interface, and with your own data services through an API.
```
Comment (justin1121) [inline README.md:135]: Do these links work?

File: README.md [hunk 3]
```diff
@@ -111,17 +110,56 @@ print(df.head())
...
+
+## About Cape Privacy and Cape Python
+
+[Cape Privacy](https://capeprivacy.com) helps teams share data and make decisions for safer and more powerful data science. Learn more at [capeprivacy.com](https://capeprivacy.com).
+
+Cape Python brings Cape's policy language to Pandas and Apache Spark. The supported techniques include tokenization with linkability as well as perturbation and rounding. You can experiment with these techniques programmatically, in Python or in human-readable policy files.
+
+### Cape architecture
+
+Cape is comprised of multiples services and libraries. You can use Cape Python as a standalone library, or you can integrate it with the Coordinator in [Cape Core](https://github.com/capeprivacy/cape/pull/336), which supports user and policy management.
```
Comment (justin1121) [inline README.md:123]: Why are we linking to this PR?

File: README.md (hunk 2 above)
Comment (StarfallProjects) [reply README.md]: fixed

**Pull request merged**